main.py

```diff
@@ -490,7 +490,53 @@ def index(self):
             p.type = point.type
             points.append(p)
         context = {'points': points}
-        return self.render('viewhighlighted.html', **context)
+        return self.render('viewhighlighteddata.html', **context)
+
+    def is_visible(self):
+        return login.current_user.is_authenticated()
+
+class ViewHighlightedMarkersMap(BaseView):
```

Comment:
I seperated the view of the data and the map of the highlighted points. The ViewHighlightedMarkersMap class is just a preperation for @SimonMand who is working on this issue. The code inside is currently not relevant - I just wanted to prepare him the view in the Admin area.
Do you want me to delete the content of the def index inside this class and just leave an empy method for now?